src/core/components/hv-doc/index.tsx

```diff
@@ -29,6 +27,9 @@ export default class HvDocState extends PureComponent<
   // </HACK>
   localDoc: Document | null = null;
 
+  // Cache elements
+  backBehaviorElementCache: Element[];
```

Comment:
minor/nitpick - if this instead stores an instance of the `BackBehaviorElements` class, we might want to rename / drop the `Cache` suffix.

Reply:
Done